praw/__init__.py

```diff
@@ -949,6 +950,19 @@ def get_popular_subreddits(self, *args, **kwargs):
         url = self.config['popular_subreddits']
         return self.get_content(url, *args, **kwargs)
 
+    def get_default_subreddits(self, *args, **kwargs):
+        """Return a get_content generator for the default subreddits.
+
+        The additional parameters are passed directly into
+        :meth:`.get_content`. Note: the `url` parameter cannot be altered.
+
+        """
+        url = self.config['default_subreddits']
+        if (len(args) < self.get_content.__code__.co_varnames.index('limit') and
+                'limit' not in kwargs):
+            kwargs['limit'] = 100
```

Comment:
Changing the default _shouldn't_ be a breaking change, however, it's possible that it could break things if people make the assumption that only 25 items are to be returned when nothing occurs. So this would probably be a change for the PRAW4 branch just in case.

Regarding the tests, they all should work as is (the username and password are provided in the tests). The cassettes need only be deleted and recreated.